internal/venice-test-common/src/main/java/com/linkedin/venice/unit/kafka/MockInMemoryAdminAdapter.java

```diff
@@ -52,7 +52,7 @@ public void createTopic(
     topicPubSubTopicConfigurationMap.put(topic, topicPubSubTopicConfiguration);
     topicPartitionNumMap.put(topic, new ArrayList<>());
     for (int i = 0; i < numPartitions; i++) {
-      topicPartitionNumMap.get(topic).add(new PubSubTopicPartitionInfo(topic, i, replication, true));
```

Comment:
Can we remove the `replication` arg from the function and everywhere else if it is not being used?